### PR TITLE
c2cupgrade improvements

### PR DESCRIPTION
**Pull request by sbrunner**
Better move message
No unneeded message
Use the unmanaged_files in the upgrade tests
Some fix

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/+dot+upgrade.yaml_tmpl [hunk 1]
```diff
@@ -36,28 +36,40 @@ files_to_move:
 files_to_remove:
 - file: '.build'
```
Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/update/+dot+upgrade.yaml_tmpl]: with quote ?

File: c2cgeoportal/scaffolds/update/+dot+upgrade.yaml_tmpl [hunk 2]
```diff
@@ -36,28 +36,40 @@ files_to_move:
...
   version: 2.3
+  from: 2.2
 - file: '.tx'
```
Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/update/+dot+upgrade.yaml_tmpl]: with quote ?

File: c2cgeoportal/scripts/c2cupgrade.py [hunk 3]
```diff
@@ -382,8 +375,12 @@ def files_to_remove(self, summary=False):
                         error = True
                         managed = True
                 if not managed:
-                    print(colorize("The file '{}' is removed.".format(file_), GREEN))
-                    if not summary:
+                    if not pre:
```
Comment (ger-benjamin) [inline c2cgeoportal/scripts/c2cupgrade.py]: this "if" can be merged with the previous one no ? (not important)

Comment (sbrunner) [reply c2cgeoportal/scripts/c2cupgrade.py]: Thanks, fixed (all) :-)

**Pull request merged**